community/modules/scheduler/cloud-batch-login-node/README.md

```diff
@@ -6,10 +6,11 @@ module.
 
 This login node:
 
-- Uses the same VM settings as the `cloud-batch-job`, such as image, machine type, etc...
-- Has the same mounted file systems as the `cloud-batch-job`.
-- Runs the same `startup-script` as the `cloud-batch-job`.
-- Contains a folder with the job template generated by the `cloud-batch-job`.
+- Uses the same VM settings as the first provided `cloud-batch-job`, such as
+  image, machine type, etc...
+- Has the same mounted file systems as the first provided `cloud-batch-job`.
```

Comment:
Does it really only mount file systems from the first?

Reply:
Not any more, updated.